azext_iot/digitaltwins/_help.py

```diff
@@ -924,7 +924,12 @@ def load_digitaltwins_help():
     helps["dt model create"] = """
         type: command
         short-summary: Uploads one or more models. When any error occurs, no models are uploaded.
-        long-summary: --models can be inline json or file path.
+        long-summary: --models can be inline json or file path. Size of input model set (ontology) is
```

Comment:
should be part of models param help not long summary. You can also specify params more than once if you do not want this absurd long help to show up everywhere

Reply:
Done